WordPress/UITests/Tests/NUXTests.swift

```diff
@@ -0,0 +1,36 @@
+import UITestsFoundation
+import XCTest
+
+class NUXTests: XCTestCase {
+    override func setUpWithError() throws {
+        try super.setUpWithError()
+        setUpTestSuite()
+
+        try LoginFlow.loginWithoutSelectingSite(email: WPUITestCredentials.testWPcomUserEmail)
+    }
+
+    override func tearDownWithError() throws {
+        try super.tearDownWithError()
+        takeScreenshotOfFailedTest()
```

Comment:
It doesn't matter in practice because, AFAIK, all these methods have empty implementations in `XCTestCase`, but from a future-proofing point of view, it's best to run the `super` tear down at the very end of the method. That is, first tear down the state that was set up in the subclass, and only afterwards tear down the state that was set in the superclass.

```suggestion
        takeScreenshotOfFailedTest()
        try super.tearDownWithError()
```